static/js/src/advantage/subscribe/react/APICalls/useFreeTrial.tsx

```diff
@@ -0,0 +1,68 @@
+import { useMutation } from "react-query";
+import {
+  postGuestFreeTrial,
+  postLoggedInFreeTrial,
+} from "../../../contracts-api";
+import useProduct from "./useProduct";
+
+import { FormValues } from "../utils/utils";
+
+const FreeTrial = () => {
```

Comment:
```suggestion
const useFreeTrial = () => {
```

Reply:
Please start custom hooks names with “use”

> Do I have to name my custom Hooks starting with “use”? Please do. This convention is very important. Without it, we wouldn’t be able to automatically check for violations of rules of Hooks because we couldn’t tell if a certain function contains calls to Hooks inside of it.
https://reactjs.org/docs/hooks-custom.html#using-a-custom-hook

Reply:
Whoops nice catch!